src/qlever/commands/example_queries.py

```diff
@@ -278,6 +409,7 @@ def execute(self, args) -> bool:
         # processing time (seconds).
         query_times = []
         result_sizes = []
+        yaml_records = {"queries": []}
```

Comment:
Give a better name, YAML is just the format and not the main thing this variable is about. Maybe `query_results`